GiTinder/Migrations/GiTinderContextModelSnapshot.cs

```diff
@@ -17,15 +16,20 @@ protected override void BuildModel(ModelBuilder modelBuilder)
 
             modelBuilder.Entity("GiTinder.Models.User", b =>
                 {
-                    b.Property<string>("UserName")
+                    b.Property<int>("Id")
```

Comment:
@Riceqrisp this should be already removed and username should be the key

Reply:
Id removed